src/Test/Integration/Model/Payment/Braintree/CreditCardTest.php

```diff
@@ -101,6 +104,10 @@ public function testAuthorizeOnlyOrder(): void
 
         $this->assertEquals(0, $authorisationTransaction->getIsClosed());
         $this->assertEquals(CreditCard::TRANSACTION_ID, $authorisationTransaction->getData('txn_id'));
+
+        /** AdditionalInformation */
+        $additionalInformation = $magentoOrder->getPayment()->getAdditionalInformation();
+        $this->assertEquals(static::PAYMENT_TITLE, $additionalInformation['method_title']);
```

Comment:
You should check for the specific credit card and not for the generic braintree cc title to find out if the mapping is working.